### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costsPerDay = 40;
```
Comment (Olexandr6) [inline src/calculateRentalCost.js:31]:
```suggestion
  const carRentCostDaily = 40;
```

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,17 @@
...
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+
+  if (days >= 7) {
+    return days * costsPerDay - sevenDaysDiscount;
+  } else if (days >= 3) {
+    return days * costsPerDay - threeDaysDiscount;
+  }
```
Comment (Olexandr6) [inline src/calculateRentalCost.js:39]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) the 3rd point

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costsPerDay = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
```
Comment (Olexandr6) [inline src/calculateRentalCost.js:33]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) the 5th point

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Pay attention to the Checklist, looks like you haven't seen it

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**